core/src/main/java/com/linecorp/armeria/client/retry/RetryRuleDecision.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client.retry;
+import static java.util.Objects.requireNonNull;
+
+import javax.annotation.Nullable;
+
+/**
+ * A {@link RetryRuleDecision} that determines whether a {@link RetryRule} retries with a {@link Backoff},
+ * skips the current {@link RetryRule} or no retries.
+ */
+public final class RetryRuleDecision {
```

Comment:
How about renaming to `RetryDecision` for brevity?

Reply:
The name is good to me. 👍